build/kyuubi-build-info

```diff
@@ -32,6 +32,7 @@ echo_build_properties() {
   echo kyuubi_trino_version="$9"
   echo user="$USER"
   echo revision=$(git rev-parse HEAD)
+  echo revision_date=$(git show -s --format=%ci HEAD)
```

Comment:
"date" is confusing since that JSR 310 uses "date" to represent `year-month-day` and "time" to represent `hour:minute:sec[.]`, how about "revision_time" or "revision_timestamp"?

Reply:
Addressed. Let's use `revision_time` for the name. Thanks for the detailed hints.